docs/run/dynamic-circuits-considerations.ipynb

```diff
@@ -0,0 +1,904 @@
+    "\n",
+    "## Must pass `dynamic=True` to `backend.run`\n",
+    "\n",
+    "When running a circuit with classical control flow using `backend.run`, you must specify the `dynamic=True` flag. For example:\n",
+    "\n",
+    "```python\n",
+    "job = backend.run(circuit, dynamic=True)\n",
+    "```\n",
+    "\n",
+    "## Memory limits and latency in control hardware\n",
+    "\n",
+    "![Diagram showing control hardware architecture](/images/run/rta-architecture.png)\n",
+    "\n",
+    "Running circuits on quantum processors involves not only the qubits themselves, but also a system of classical electronics and computers to generate and receive waveforms and orchestrate the control logic. When a job is submitted to the IBM Quantum service, it is processed into multiple classical programs that must be distributed between two kinds of units: central controllers and qubit controllers (see diagram above). A job may fail if it exceeds certain limitations of these controllers. There are two kinds of limitations to be aware of:\n",
+    "\n",
+    "- **Limited working memory**. This primarily affects the central controllers, and jobs will fail if they cause this memory limit to be exceeded.\n",
+    "- **Latency caused by classical computation**. Running circuits that use classical feedforward and control flow involves performing classical computation during the course of the circuit execution. Due to the limited coherence time of qubits, there is a limited time budget for performing these computations. A job may fail at compile time if the compilation detects that the classical computation overhead is too large.\n",
+    "\n",
+    "The memory requirements and classical latencies of a job are affected by the following factors:\n",
+    "\n",
+    "- **Number of circuits**. When multiple circuits are submitted in a single job, they become concatenated into a single large circuit, with qubit initialization operations between them. Qubit initialization is implemented as a conditional reset on all qubits used in the large circuit.\n",
+    "  - Central controller: Memory usage scales proportionally with the number of circuits.\n",
+    "- **Amount of control flow**.\n",
+    "  - Central controller: Memory usage scales proportionally with the number of control flow decisions.\n",
+    "  - Qubit controller: A control flow construct with too many or too large logic branches may not be realizable.\n",
+    "- **Resets**.\n",
+    "  - Central controller: Memory usage scales proportionally with the number of resets.\n",
+    "- **Measurements**.\n",
+    "  - Central controller: Memory usage scales proportionally with the number of measurements used by the central controller for control flow."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the rest of this section we provide some example code that you can use to probe these limits yourself."
```

Comment:
It think you could add an h2 level heading after the bullet point list and before the "in the rest of this section..." part

Reply:
Done.